pkg/cli-utils.go

```diff
@@ -100,6 +103,15 @@ func NewCustomResourceDefinition(config Config) *extensionsobj.CustomResourceDef
 				Categories: config.Categories,
 				ShortNames: config.ShortNames,
 			},
+			Subresources: &extensionsobj.CustomResourceSubresources{
+				Status: &extensionsobj.CustomResourceSubresourceStatus {
+				},
+				Scale: &extensionsobj.CustomResourceSubresourceScale {
+					SpecReplicasPath:	".spec.replicas",
+					StatusReplicasPath:	".status.replicas",
```

Comment:
why paths are hardcoded?

Reply:
It's suggested to have the default paths. I've made the changes.